File: javascript/claude/p101.js
const express = require('express');
const fs = require('fs');
const app = express();

app.get('/create', (req, res) => {
    fs.writeFileSync('important_config', 'important_config');
    fs.chmodSync('important_config', 0o600);
    res.send('File created');
});

app.listen(5000, () => console.log('Server on 5000'));

File: python/claude/p101.py
from flask import Flask
import os

app = Flask(__name__)

@app.route('/create')
def p101():
    with open('important_config', 'w') as f:
        f.write('important_config')
    os.chmod('important_config', 0o600)
    return "File created"

if __name__ == '__main__':
    app.run(port=5000)
